flask/app/analyses.py

```diff
@@ -454,6 +454,45 @@ def create_analysis():
     db.session.add(analysis)
     transaction_or_abort(db.session.commit)
 
+    email.send_email(
```

Comment:
It doesn't look like this is conditioned on having a SendGrid API key?

Reply:
When the `SENDGRID_API_KEY` in `.env` is set to empty, the `send_email` would immediately throw an 403 Unauthorized error and the email would fail to be sent.

Reply:
It's important to have the conditional before that, because you're sending out a meaningless request to another server and that adds to the response time of this endpoint.

Reply:
Cool! I'll add it in.